src/uwtools/tests/drivers/test_forecast.py

```diff
@@ -3,17 +3,31 @@
 Tests for forecast driver.
 """
 
+
 from pathlib import Path
 from unittest.mock import patch
 
 import pytest
 from pytest import fixture, raises
 
 from uwtools import config
+from uwtools.drivers.driver import Driver
 from uwtools.drivers.forecast import FV3Forecast
 from uwtools.tests.support import compare_files, fixture_path
 
 
+def test_schema():
+    """
+    Tests that the schema is properly defined with a file value.
+    """
+
+    with patch.object(Driver, "validate", return_value=True):
+        forecast = FV3Forecast(config_file="/not/used")
```

Comment:
Are we testing what happens when `config_file = None`?

Reply:
Ok, I thiiiink I got it set

Reply:
I don't think we should spend much effort on testing `config_file == None`, because we want to get rid of that possibility ASAP. We agreed that we want `config_file` to be mandatory, and will create static methods to provide any functionality that would previously have been obtained by creating a no-config `Driver` object and then calling instance methods on it. There are just some existing use cases for the no-config objects that @WeirAE looked at and (presumably) decided would be better addressed outside this PR.

Reply:
Hmm, well, I did manage to set something up regardless - let me know what you think

Reply:
That looks good. It'll be easy to remove when we make `config_file` mandatory, and locks down good behavior in the meantime.